tgui/packages/tgui/interfaces/ChemRequestReceiver.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @file
+ * @copyright 2022
+ * @author LeahTheTech (https://github.com/TobleroneSwordfish)
+ * @license MIT
+ */
+
+import { useBackend, useLocalState, useSharedState } from '../backend';
+import { Button, LabeledList, Input, Section, Flex, Box, Stack, Tabs } from '../components';
+import { Window } from '../layouts';
+
+import { capitalize } from '../../common/string';
+
+const ChemRequest = (props, context) => {
+  const { act } = useBackend(context);
+  const {
+    name,
+    id,
+    reagent_name,
+    reagent_color,
+    volume,
+    notes,
+    area,
+    state,
+    interactable,
+    age,
+  } = props;
+  // some mild colour hackery to make the text visible on (hopefully) any reagent colour
+  const color_string = "rgba(" + reagent_color[0] + "," + reagent_color[1] + ", " + reagent_color[2] + ", 1)";
+  const lightness = reagent_color.reduce((a, b) => a + b, 0)/3;
+  return (
+    <Section height>
```

Comment:
`height` is not a boolean prop, but a number prob. This can probably be removed.